source/jormungandr/jormungandr/scenarios/distributed.py

```diff
@@ -273,6 +275,93 @@ def finalise_journeys(self, future_manager, request, responses, context, instanc
             journeys=journeys_to_complete,
         )
 
+    def _compute_isochrone(self, future_manager, request, instance, krakens_call):
+        logger = logging.getLogger(__name__)
+        logger.debug('request datetime: %s', request['datetime'])
+
+        isochrone_center = request['origin'] if request['origin'] else request['destination']
+        requested_modes = (
+            {mode for mode, _, _ in krakens_call} if request['origin'] else {mode for _, mode, _ in krakens_call}
+        )
+
+        logger.debug('requesting places by uri orig: %s', isochrone_center)
+
+        requested_orig = PlaceByUri(future_manager=future_manager, instance=instance, uri=isochrone_center)
+
+        requested_orig_obj = get_entry_point_or_raise(requested_orig, isochrone_center)
+
+        direct_paths_by_mode = {}
+
+        orig_proximities_by_crowfly = ProximitiesByCrowflyPool(
+            future_manager=future_manager,
+            instance=instance,
+            requested_place_obj=requested_orig_obj,
+            modes=requested_modes,
+            request=request,
+            direct_paths_by_mode=direct_paths_by_mode,
+            max_nb_crowfly_by_mode=request['max_nb_crowfly_by_mode'],
+        )
+
+        orig_places_free_access = PlacesFreeAccess(
+            future_manager=future_manager, instance=instance, requested_place_obj=requested_orig_obj
+        )
+
+        direct_path_type = (
+            StreetNetworkPathType.BEGINNING_FALLBACK
+            if request['origin']
+            else StreetNetworkPathType.ENDING_FALLBACK
+        )
+        orig_fallback_durations_pool = FallbackDurationsPool(
+            future_manager=future_manager,
+            instance=instance,
+            requested_place_obj=requested_orig_obj,
+            modes=requested_modes,
+            proximities_by_crowfly_pool=orig_proximities_by_crowfly,
+            places_free_access=orig_places_free_access,
+            direct_paths_by_mode=direct_paths_by_mode,
+            request=request,
+            direct_path_type=direct_path_type,
```

Comment:
if i understand well, this argument is used to tell in which way(aka. clockwise) the matrix should be computed right?

perhaps we should find a name more accurate... let's talk about that later

Reply:
Yes exactly !